### DISCUSSION TITLE
WordPress to Manifold Journals migration

### DISCUSSION THREAD

**Discussion by jojokarlin** [ManifoldScholar/manifold#3097] [General]
Hi Manifold! 
As we prepare to migrate a WordPress journal to Manifold, many questions are coming up and we wanted to put them somewhere.
We've begun with an XML export of the current site and converted the files to Markdown using https://github.com/lonekorean/wordpress-export-to-markdown.

We're thinking ahead (in no particular order) to:

- how best to prep these blog posts for eventual ingestion
- how to handle url redirects
- how Manifold will serve our regular short forms (which currently live on separate pages from issues and get updated on a rolling basis)
- how to batch load assets (and where)
- how much we can prep before ingestion
- how to manage the quality checks after ingestion

A couple specific questions:
1. how will the journal layout handle breadcrumbs and meta data for journal issues?
2.  can we bulk categorize texts, or include categories in manifest ingestion?

Our fearless website committee (@walshbr, @lwaltzer, @hamnet70) will have many questions and we will add them to this discussion as they come up.

Thanks as always!
Jojo

**Comment by jojokarlin** [comment]
Just chatted with Terence about some of these issues.
- workflow! We're thinking it makes sense to do author submissions in Word, use a Word template for styles, use Mammoth locally (one or two editors) to convert all the files to HTML, have stagers review and tidy the HTML, zip HTML with CSS and images to load to Manifold. Terence recommends pulling Scribenet.com Word Template and CSS and SAI Lite tool for loading template in Word (https://scribenet.com/wfdw/resources)
- short forms brought into the Journal collection as separate project
- we can't bulk categorize texts

More to come!

**Comment by zdavis** [comment]
Closing this, as I think this migration happened long ago!
